runtime/src/main/java/com/tns/AppConfig.java

```diff
@@ -18,20 +18,7 @@
         MemoryCheckInterval("memoryCheckInterval", 0),
         FreeMemoryRatio("freeMemoryRatio", 0.0),
         Profiling("profiling", ""),
-        MarkingMode("markingMode", "Full");
-
-        public static final KnownKeys[] asArray = {
-            V8FlagsKey,
-            CodeCacheKey,
-            HeapSnapshotScriptKey,
-            SnapshotFile,
-            ProfilerOutputDirKey,
-            GcThrottleTime,
-            MemoryCheckInterval,
-            FreeMemoryRatio,
-            Profiling,
-            MarkingMode
-        };
+        MarkingMode("markingMode", com.tns.MarkingMode.values()[0]);
```

Comment:
`com.tns.MarkingMode.full` as default value instead of `com.tns.MarkingMode.values()[0]`